docker/README.md

```diff
@@ -6,10 +6,10 @@ Default password is `admin`.
 
 ## Example Runs
 ```sh
-docker run --network="host"  -p 8888:8888 -t graph-notebook
+docker run -p 8888:8888 -t graph-notebook
```

Comment:
Do you know if there was any reason they added this `--network="host"` option as their example? Or is it potentially leftover code from older environments that used this flag?

Reply:
Looks we can add some explanation in the documentation around this? https://docs.docker.com/network/drivers/host/